### DISCUSSION TITLE
How to change mastercontainer variables with Quadlet (rootless podman systemd)?

### DISCUSSION THREAD

**Discussion by PhysicsIsAwesome** [nextcloud/all-in-one#5745] [Questions] [answered]
Hey,

I would like to change a few variables on an existing Nextcloud AIO with rootless Quadlet on Fedora server 40, specifically NEXTCLOUD_DATADIR and AIO_DISABLE_BACKUP_SECTION. Steps I took:

1. Stop all NC containers
2. Stop mastercontainer via `systemctl --user stop nextcloud-aio-mastercontainer`
3. Copied all files from NEXTCLOUD_DATADIR to the new location.
4. Changed NEXTCLOUD_DATADIR in `nextcloud-aio-mastercontainer.container` file
5. ` systemctl --user daemon-reload`
6. `systemctl --user start nextcloud-aio-mastercontainer`
7. Start all other NC containers
8. Created a new file in NC and checked to which location it was saved.
9. Realized that the new file was created in the old storage location instead of the new

Does anyone know how to solve this?

**Comment by Extarys** [comment, marked as answer]
I believe you need to remove the container `podman stop nextcloud-aio-mastercontainer && podman rm nextcloud-aio-mastercontainer` and then start the service again for the changes to apply.
